src/frontend/components/SchemaForm/_RenderFormInput.tsx

```diff
@@ -70,13 +74,37 @@ export function RenderFormInput(props: IRenderFormInputProps) {
 
   switch (type) {
     case "email":
-    case "password":
     case "url":
     case "color":
       return <FormInput type={type} {...formProps} />;
     case "number":
       return <FormNumberInput {...formProps} />;
-
+    case "password":
+      return (
```

Comment:
Can you move all the logic here to the new password component
Like here it should just be
```
   return <FormPasswordInput {...formProps} />;
```
everything else should be in the component

Reply:
Got It